src/v2/Components/NavBar/menuData.ts

```diff
@@ -155,16 +155,16 @@ export const ARTWORKS_SUBMENU_DATA: MenuLinkData = {
         href: "/gene/trove",
```

Comment:
**question**: should we change "Trovo" to "Trove: Editor’s Picks"?

<img width="1048" alt="FX-3930  Update Navigation Links - Jira 2022-05-04 13-50-58" src="https://user-images.githubusercontent.com/3513494/166667910-a763d84e-d49f-45d6-8930-6ac1b9d8d80b.png">

Reply:
Good catch @dimatretyak! Yes, I think I missed that